independent-publisher-connectors/Proposal - GIPHY (Independent Publisher)/apiDefinition.swagger.json

```diff
@@ -0,0 +1,878 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "GIPHY",
+    "description": "This custom connector wraps around GIPHY. It can be used to get GIFs related to a search string that is provided as input.",
+    "version": "1.0",
+    "contact": {
+    "name": "Priyanshu Srivastav",

```

Comment:
@priyanshu870 please fix the indentation. all files should be properly formatted.